test/selenium/configurations.rb

```diff
@@ -127,6 +127,12 @@ def configure_for_chrome_web(args)
   common_configuration_for_web(args)
 end
 
+def configure_for_safari_web(args)
+  common_configuration_for_web(args)
+  # Do not block pop-up windows
+  `defaults write com.apple.Safari com.apple.Safari.ContentPageGroupIdentifier.WebKit2JavaScriptCanOpenWindowsAutomatically -bool true`
```

Comment:
I don't understand how this command is changing the safari setting. It is fine as-is, but I want to know how it works.